atlp-rwanda/falcons-ec-be
# #184637235 Integrate coveralls code coverage and circleCi badge with readme

**What does this PR do?**

- Integrate coverage and circleCI test badges with readme

**Description of Task to be completed?**

- Testing api using appropriate tools
- Uploading tests to circleCi and coveralls
- integrating badges to a readme file 

**How should this be manually tested?**

- clone the project and use the terminal to start the development server by "npm run dev"
- make sure that you have updated version of coveralls, jest and PostgreSQL installed
- create a .test file, database, and
- then use the database name and password to make the DEV_DATABASE_URL in your env file,
- refer to the env.example file

**Any background context you want to provide?**
none

**What are the relevant pivotal tracker/Trello stories?**

- Tracker: 184637234
- Tracker: 184637234

**Screenshots (if appropriate)**
none

**Questions:**
none
